.config/.terraform-docs.yml

```diff
@@ -0,0 +1,90 @@
+version: ""
+
+formatter: markdown table
+
+name: terraform-aws-terraform-state-s3-bucket-centralized-with-roles
+
+content: |-
+  # terraform-state-s3-bucket-centralized-with-roles
+
+  This is intended to be used by an organization for all of their own accounts. This does not protect access to DynamoDB locking of other accounts, it only restricts access S3 paths for each account.
+  
+  This restriction is put in place by creating a unique role for each account, then attaching an assumerole policy that trusts the corresponding account to assume it. You will still need to give permission to assume roles to your users/roles that are used to apply terraform in other accounts, and configure your state appropriately to use this.
+  
+  As of v3.0, all public access is blocked by default.  There are individual parameters which can be set to "false" if public bucket/state access is desired:
+  * block_public_acls
+  * block_public_policy
+  * ignore_public_acls
+  * restrict_public_buckets
+  
+  ## Example Config:
+
+  ```hcl
+  {{ include "examples/with_trusted_accounts.tf" }}
+  ```
+  
+  ## Example Backend Config:
+
+  ```hcl
+  {{ include "examples/backend_config.tf" }}
+  ```
+
+  ## Example to Initialize the backend:
+  ```
+  terraform init -backend-config="access_key=AKIAIOIEXAMPLEOXJA" -backend-config="secret_key=PiNMLcNOTAREALKEYGQGzz20v3"
+  ```
```

Comment:
I assume this are outdated but i need to ask, is it ok to post access key and secret key on the documentation?

Reply:
These have been in place for...a very long time.  Good catch; I'll make them something generic, and I'll ask Ivan if we need to pursue this further.

Reply:
Do we need that example? Normally we just leave it as `terraform init -backend-config=` and even then this is a module so it doesn't need this information?

Reply:
I like it tough, like documentation that is thorough

Reply:
For now I'll mark this resolved. If we want to remove the example, we can create an issue for it.